cypress/helpers/DbHelpers.js

```diff
@@ -7,146 +7,152 @@ import { patch } from "../../api/models/requests/patchModel";
 import { cautionaryAlert } from "../../api/models/requests/cautionaryAlertModel";
 import { createCautionaryAlert } from "../../api/cautionary-alert";
 import { tenureToPersonTenure, tenureToAssetTenure } from "./helpers";
-
-export const addTestRecordToDatabase = (dbTableName, testDbRecord) => {
-  cy.log("Seeding database").then(async () => {
-    cy.log(
-      `Adding test record to database table ${dbTableName} and creating a record of it in recordsToDelete.json file`
-    );
-    return new Cypress.Promise((resolve) => {
-      DynamoDb.createRecord(dbTableName, testDbRecord).then(() => {
-        resolve();
-      });
-    }).then(() => {
-      cy.log("Database seeded!");
+import { changeOfName } from "../../api/models/requests/processModel";
+
+export const addTestRecordToDatabase = (dbTableName, testDbRecord, testRecordKey) => {
+    cy.log("Seeding database").then(async () => {
+        cy.log(
+            `Adding test record to database table ${dbTableName} and creating a record of it in recordsToDelete.json file`
+        );
+        return new Cypress.Promise((resolve) => {
+            DynamoDb.createRecord(dbTableName, testDbRecord, testRecordKey).then(() => {
```

Comment:
Why not:
```ts
DynamoDb.createRecord(dbTableName, testDbRecord, testDbRecord.id)
```

Reply:
Because the key might be composite, e.g. `{ id: xxx, targetId: xxx }`. There's no way to determine what the key is without manually coding it.

Reply:
Not sure composite keys make sense given the entities we're dealing with.
I'd guess that the targetId would be more like foreign key in our case.

I'm assuming you need the targetId to clean up related resources. Like: if you're cleaning up tenure, you also want to clean up an asset?

You could probably do a check of: "does entity have targetId? If yes, clean up that entity as well.", but I won't press  for these changes given the nature of this PR isn't about cleaning up test utils.